src/main/java/ui/components/PanelMenuBarWithRename.java

```diff
@@ -0,0 +1,105 @@
+package ui.components;
+import backend.interfaces.IModel;
+import javafx.scene.control.Label;
+import ui.issuepanel.FilterPanel;
+import javafx.application.Platform;
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.HBox;
+
+/**
+ * Created by manmeetsinghsaluja on 23/11/15.
+ */
```

Comment:
We don't usually show author name in the header comment. Write a proper header comment for this class.